_ml-commons-plugin/extensibility/connectors.md

```diff
@@ -1,135 +1,44 @@
-
-### Adding trusted endpoints
-
-To configure connectors in OpenSearch, add the trusted endpoints to your cluster settings using the `plugins.ml_commons.trusted_connector_endpoints_regex` setting, which supports Java regex expressions, as shown in the following example:
-
-```json
-PUT /_cluster/settings
-{
-    "persistent": {
-        "plugins.ml_commons.trusted_connector_endpoints_regex": [
-            "^https://runtime\\.sagemaker\\..*\\.amazonaws\\.com/.*$",
-            "^https://api\\.openai\\.com/.*$",
-            "^https://api\\.cohere\\.ai/.*$",
-            "^https://bedrock\\..*\\.amazonaws.com/.*$"
-        ]
-    }
-}
-```
-{% include copy-curl.html %}
-
-### Enabling ML nodes
-
-Most connectors require the use of dedicated ML nodes. To make sure you have ML nodes enabled, update the following cluster settings:
-
-```json
-PUT /_cluster/settings
-{
-    "persistent": {
-        "plugins.ml_commons.only_run_on_ml_node": true,
-    }
-}
-```
-{% include copy-curl.html %}
-
-If you are running a remote inference or local model, you can set `"plugins.ml_commons.only_run_on_ml_node"` to `false` and use data nodes instead.
-
-
-### Setting up connector access control
-
-To enable access control on the connector API, use the following cluster setting:
-
-```json
-PUT /_cluster/settings
-{
-    "persistent": {
-        "plugins.ml_commons.connector_access_control_enabled": true
-    }
-}
-```
-{% include copy-curl.html %}
+You can provision connectors in two ways:
 
-When enabled, the `backend_roles`, `add_all_backend_roles`, or `access_model` options are required in order to use the connector API. If successful, OpenSearch returns the following response:
+1. An [external connector](#external-connector), saved in a connector index, which can be reused and shared with multiple remote models but requires access to both the model, the connector inside of OpenSearch, and the third party being accessed by the connector, such as OpenAI or SageMaker.
 
-```json
-{
-  "acknowledged": true,
-  "persistent": {
-    "plugins": {
-      "ml_commons": {
-        "connector_access_control_enabled": "true"
-      }
-    }
-  },
-  "transient": {}
-}
-```
+2. An [local connector](#local-connector), saved in the model index, can only be used with one remote model. Unlike a standalone connector, users only need access to the model itself to access an internal connector because the connection is established inside the model.
 
-## Creating a connector
 
-You can build connectors in two ways:
+## Supported connectors
 
-1. A **standalone connector**, saved in a connector index, can be reused and shared with multiple remote models but requires access to both the model and the third party being accessed by the connector, such as OpenAI.
+As of OpenSearch 2.9, connectors have been tested for the following ML tools, though it is possible to create connectors for other tools not listed here:
```

Comment:
Should we call Amazon Sagemaker, OpenAI and Cohere as "ML tools" ?  I think we should say ML services or platform

Reply:
```suggestion
As of OpenSearch 2.9, connectors have been tested for the following ML services, though it is possible to create connectors for other platforms not listed here:
```